src/librbd/ImageCtx.cc

```diff
@@ -930,7 +930,8 @@ struct C_InvalidateCache : public Context {
         "rbd_journal_object_flush_age", false)(
         "rbd_journal_pool", false)(
         "rbd_journal_max_payload_bytes", false)(
-        "rbd_journal_max_concurrent_object_sets", false);
+        "rbd_journal_max_concurrent_object_sets", false)(
+        "rbd_mirror_resync_after_disconnect", false);
```

Comment:
No need to treat this as a librbd image option -- just use it from rbd-mirror daemon only